Lombiq.NodeJs.Extensions.Samples/Lombiq.NodeJs.Extensions.Samples.csproj

```diff
@@ -1,29 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
+  <Import Project="..\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.props" />
+
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**</DefaultItemExcludes>
+    <NodeJsExtensionsStylesSourceFolder>NonDefaultScssFolder</NodeJsExtensionsStylesSourceFolder>
+    <NodeJsExtensionsStylesTargetFolder>wwwwNonDefaultCssFolder</NodeJsExtensionsStylesTargetFolder>
```

Comment:
Add docs about such configs.

Reply:
Would you put that documentation into the main _Readme.md_ or into _Docs/Styles.md_? It is specific for styles, but I'm wondering if it's not important enough to be in the main readme.

Reply:
Into Styles.